inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -4,59 +4,62 @@
 
 #include <limits>
 #include <algorithm>
-
 #include <string>
 #include <map>
 #include <vector>
 #include <iostream>
 #include <cmath>
 #include <tuple>
 #include <cctype>
+#include <memory>
 
+#include <cnn_network_ngraph_impl.hpp>
```

Comment:
must not be used in plugins

Reply:
let's continue in https://github.com/openvinotoolkit/openvino/pull/2506#discussion_r516113662